prime-router/metadata/fhir_mapping/hl7/datatype/Annotation.yml

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
I have been removing these headers when I copy the file.  Not sure it matters, but maybe remove?